wolfssl/internal.h

```diff
@@ -2777,7 +2777,7 @@ struct WOLFSSL_CTX {
     byte            readAhead;
     void*           userPRFArg; /* passed to prf callback */
 #endif
-#ifdef HAVE_EX_DATA
+#if defined(HAVE_EX_DATA) || defined(FORTRESS)
```

Comment:
Isn't it simpler to have FORTRESS enable HAVE_EX_DATA in settings if not already on?

Reply:
Done!

Removed Fortress Conditional and added check in settings.h to set HAVE_EX_DATA if not already defined when FORTRESS is set.

```
#ifdef FORTRESS
     #ifndef HAVE_EX_DATA
         #define HAVE_EX_DATA
     #endif
...
```